webui/api-ts-sdk/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@determined-ai/api-ts-sdk",
+  "version": "1.0.0",
```

Comment:
blocking: Can we make sure we've considered what we want our numbering scheme to look like here? For example, should it map directly to our API version? (If so, maybe we start at 0.0.0 for the moment, until we're ready to declare API v1 "done"?)

Reply:
sidenote the version probably wouldn't really matter until we start publishing the package on NPM. could probably defer it until we come up with fine grain versioning for our APIs

Reply:
I updated the version